### DISCUSSION TITLE
Guidance on Radiative Transfer Simulations for the Martian Atmosphere (300-700 GHz)

### DISCUSSION THREAD

**Discussion by jayeshgit65** [atmtools/arts#899] [Q&A]
Dear ARTS Community,

I hope this message finds you well.

I am currently working with PyARTS 2.6.12 on radiative transfer simulations for the Martian atmosphere within the frequency range of 300 to 700 GHz. While exploring the provided control file examples, I noticed that most examples are tailored to Earth's atmosphere. Unfortunately, I couldn't find specific examples or references that align closely with Martian atmospheric profiles and conditions.

To better understand and adapt PyARTS for Martian studies, I would greatly appreciate:

1)Example setups: Control files or configurations that have been successfully used for RT simulations specific to Mars.
2)Guidance on atmospheric profiles: Information on constructing or adapting profiles (e.g., pressure, temperature, and gas composition) for Martian RT simulations in different configurations.

I have explored the ARTS documentation and available literature but am still unsure about the best approach to adapt these tools for the unique characteristics of the Martian atmosphere. Your insights or references to relevant scripts, datasets, or control files would be invaluable.

Thank you for your time and support. Your expertise can significantly aid my understanding and contribute to advancing RT studies for planetary atmospheres.

Warm regards,
Jayesh

**Comment by carolinecox** [comment]
Hi Jayesh,
I am an ARTS user, not part of the dev team, but I have been using ARTS 2.2 and I remember seeing references to Mars examples and perhaps some example atm profiles which may help you.  You can get hold off the ARTS2.2 package online still.  
Caroline

**Reply by jayeshgit65** [reply]
Thank you for your suggestion, @carolinecox !
